packages/template-retail-react-app/package.json

```diff
@@ -18,15 +18,19 @@
         "@lhci/cli": "^0.9.0",
         "@loadable/component": "^5.15.3",
         "@peculiar/webcrypto": "^1.4.2",
+        "@salesforce/commerce-sdk-react": "file:../../../pwa-kit/packages/commerce-sdk-react/src",
+        "@tanstack/react-query": "4.28.0",
         "@testing-library/dom": "^9.0.1",
         "@testing-library/jest-dom": "^5.16.5",
-        "@testing-library/react": "^13.4.0",
+        "@testing-library/react": "^12.1.5",
         "@testing-library/react-hooks": "^8.0.1",
         "@testing-library/user-event": "^14.4.3",
+        "ajv": "^8.17.1",
+        "ajv-keywords": "^5.1.0",
         "base64-arraybuffer": "^0.2.0",
         "bundlesize2": "^0.0.31",
         "card-validator": "^8.1.1",
-        "commerce-sdk-isomorphic": "^1.12.0",
+        "commerce-sdk-isomorphic": "^3.3.0",
```

Comment:
Some packages like this are aligned what's found in the commerce-sdk-react package.